node/src/reactor.rs

```diff
@@ -552,50 +571,56 @@ where
             QUEUE_DUMP_REQUESTED.store(false, Ordering::SeqCst);
         }
 
-        let (event, queue) = self.scheduler.pop().await;
+        let ((ancestor, event), queue) = self.scheduler.pop().await;
+        trace!(%event, %queue, "current");
 
         // Create another span for tracing the processing of one event.
```

Comment:
We already get a new span every time `crank` is called, as the `instrument` macro will create one for us. But instead of creating another nested span, we made room for an event ID on the parent.